README.md

```diff
@@ -0,0 +1,128 @@
+# Book Library API
+
+This is a Book Library that was created as a project on the Command Shift Bootcamp.
+
+The aim was to create a REST API that allows a user to create Readers, Books, Genres and Authors, to update these, and also search by Authors and Genres to return a list of books related to them.
+
+PostgreSQL was used as the database language for this project.
+
+A TDD approach was taken with this project.
+
+## Tech & Frameworks
+
+This project uses Node.js, Express.js, and Sequelize. A combination of Mocha, Chai, and Supertest were used for testing the code. I also carried out a small amount of manual testing using Postman and PgAdmin to ensure changes were happening in the database.
+
+## Installation
+
+`npm install` should install everything required for the project once the repo has been cloned
+
+## API
+
+### Authors
+
+#### GET
```

Comment:
Theres some good detail here, if you want to improve the look of it a bit more, you could update this section so its all in a markdown table, something like:
```
### Authors

| Route | HTTP Verb | Description |
| ----------- | ----------- | ----------- |
| `/authors` | GET | Returns a list of authors | 
| `/authors/:author` | GET | Returns a specific author |
```
Theres more info on extended formatting rules for READMEs available [here](https://www.markdownguide.org/cheat-sheet/#extended-syntax)